examples/using-mobx/src/components/layout.js

```diff
@@ -1,32 +1,32 @@
-import React, { Component } from "react"
+import React from "react"
+import PropTypes from "prop-types"
 import { Link } from "gatsby"
 import Counter from "./Counter"
 
-import DevTools from "mobx-react-devtools"
-
-class DefaultLayout extends Component {
-  render() {
-    return (
-      <div>
-        <Link to="/">
-          <h3>MobX example</h3>
-        </Link>
-        <DevTools />
-        <Counter />
-        <ul>
-          <li>
-            <Link to="/a/">a</Link>
-          </li>
-          <li>
-            <Link to="/b/">b</Link>
-          </li>
-          <li>
-            <Link to="/c/">c</Link>
-          </li>
-        </ul>
-        {this.props.children}
-      </div>
-    )
-  }
+const DefaultLayout = props => {
+  const { children } = props
+  return (
+    <div>
+      <Link to="/">
+        <h3>MobX example</h3>
+      </Link>
+      <Counter />
+      <ul>
+        <li>
+          <Link to="/a/">a</Link>
+        </li>
+        <li>
+          <Link to="/b/">b</Link>
+        </li>
+        <li>
+          <Link to="/c/">c</Link>
+        </li>
+      </ul>
+      {children}
+    </div>
+  )
+}
+DefaultLayout.propTypes = {
+  children: PropTypes.object.isRequired, // eslint-disable-line
```

Comment:
Could you explain why you need the eslint-disable-line?

Reply:
when the example was first created, running the lint task would complain about this line.